### DISCUSSION TITLE
QFIELD IOS

### DISCUSSION THREAD

**Discussion by comriegos** [opengisch/QField#4473] [Q&A]
Buenos días, 
soy nueva usando Qfield, la cuestión es que estamos planteándonos en la empresa la compra de una tablet para poder funcionar fácil y de forma práctica cuando estemos en campo. No obstante, he visto que actualmente la versión que hay de qfield para iOs es una beta y al probarla el fin de semana, me estaba dando problemas. Pues, cuando hacia alguna modificación o un dibujo se quedaba cargando y se me cerraba la aplicación. Y así todo el rato. Entonces, me gustaría saber cual es la experiencia de ustedes con el iPad, si lo soléis trabajar o en casos contrarios cuál es la que mejor va para esta aplicación. 
Muchas gracias de antemano!

**Comment by BTreeTech** [comment]
I have done basic testing on an iPad Mini and it worked ok. We have found cheaper 7 inch android tablets don't have the performance required to give a smooth interaction.

**Reply by comriegos** [reply]
Perfecto, muchas gracias!
